diff --git a/node_modules/babel-eslint/lib/analyze-scope.js b/node_modules/babel-eslint/lib/analyze-scope.js
index 80bdff0..e2128c0 100644
--- a/node_modules/babel-eslint/lib/analyze-scope.js
+++ b/node_modules/babel-eslint/lib/analyze-scope.js
@@ -1,14 +1,13 @@
 "use strict";
 
 const t = require("@babel/types");
-const requireFromESLint = require("./require-from-eslint");
 
-const escope = requireFromESLint("eslint-scope");
-const Definition = requireFromESLint("eslint-scope/lib/definition").Definition;
-const OriginalPatternVisitor = requireFromESLint(
+const escope = require("eslint-scope");
+const Definition = require("eslint-scope/lib/definition").Definition;
+const OriginalPatternVisitor = require(
   "eslint-scope/lib/pattern-visitor"
 );
-const OriginalReferencer = requireFromESLint("eslint-scope/lib/referencer");
+const OriginalReferencer = require("eslint-scope/lib/referencer");
 const fallback = require("eslint-visitor-keys").getKeys;
 const childVisitorKeys = require("./visitor-keys");
 
